processor/resourcedetectionprocessor/README.md

```diff
@@ -426,6 +426,28 @@ processors:
     override: false
 ```
 
+### Kubeadm Metadata
+
+Queries the K8S API server to retrieve the following resource attributes:
+
+    * k8s.cluster.name
+
+The following permissions are required:
+```yaml
+kind: Role
+metadata:
+  name: otel-collector
```

Comment:
Shouldn't the namespace be set to `kube-system` here since the `kubeadm-config` belongs to that namespace? (sorry my previous example was just to illustrate the idea).

Also I would suggest we include the `RoleBinding` here as well to make clear how this cross-namespace access should be defined (Collector runs in `default` namespace usually).